3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -84,7 +84,12 @@ function setup_primary() {
     local config="{_id: \"${MONGODB_REPLICA_NAME}\", members: [ {_id: 0, host: \"${member_host}\"} ]}"
 
     info "Initiating MongoDB replica using: ${config}"
-    mongo admin --eval "rs.initiate(${config});${WAIT_PRIMARY_STATUS}" --quiet
+    mongo admin --eval "
+      if (db.isMaster().setName != \"${MONGODB_REPLICA_NAME}\") {
+        rs.initiate(${config});
+        ${WAIT_PRIMARY_STATUS}
+      }
+    " --quiet
```

Comment:
Part of the logs after a restart:

```
2016-10-05T20:11:49.518+0000 I REPL     [initandlisten] Did not find local voted for document at startup;  NoMatchingDocument: Did not find replica set lastVote document in local.replset.election
2016-10-05T20:11:49.518+0000 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2016-10-05T20:11:49.518+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/var/lib/mongodb/data/diagnostic.data'
2016-10-05T20:11:49.519+0000 I NETWORK  [HostnameCanonicalizationWorker] Starting hostname canonicalization worker
2016-10-05T20:11:49.605+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2016-10-05T20:11:51.031+0000 I ACCESS   [conn1] note: no users configured in admin.system.users, allowing localhost access
=> [Wed Oct 05 20:11:52] Initiating MongoDB replica using: {_id: "rs0", members: [ {_id: 0, host: "mongodb-0.mongodb.demo.svc.cluster.local"} ]}
```

Replication data was not found, replica set created from scratch.